### PR TITLE
Fixes anti-drops implants not working

### LINKED ISSUE #23305: No-Drop Implant fails to release weapon once it's disabled
Captain ahelped becuase they couldn't drop their weapon even after they disabled their no-drop implant. 

What should happen: The weapon should be able to be placed into a backpack, dropped, thrown, or placed into a recharger once the implant is disabled.

### PR DESCRIPTION
**Pull request by lzimann**
Fixes #23305
The only time they were making it NODROP was when you would deactivate the implant.

### CODE REVIEW

File: code/modules/surgery/organs/augments_internal.dm [hunk 1]
```diff
@@ -83,7 +86,7 @@
 
 /obj/item/organ/cyberimp/brain/anti_drop/proc/release_items()
 	for(var/obj/item/I in stored_items)
-		I.flags ^= NODROP
```
Comment (lzimann) [inline code/modules/surgery/organs/augments_internal.dm]: guess i could've made both activate/deactivate use the release_items proc(and rename it)

**Comment by XDTM** [comment]
damn i was fixing it too

File: code/modules/surgery/organs/augments_internal.dm [hunk 2]
```diff
@@ -72,7 +74,8 @@
 		return
 	var/range = severity ? 10 : 5
```
Comment (XDTM) [inline code/modules/surgery/organs/augments_internal.dm]: Won't this always be 10? If i remember correctly severity scales from 3 to 1

Comment (lzimann) [inline code/modules/surgery/organs/augments_internal.dm]: i have no idea

File: code/modules/surgery/organs/augments_internal.dm (hunk 1 above)
Comment (xxalpha) [inline code/modules/surgery/organs/augments_internal.dm]: This is an XOR operation, which means it was supposed to be toggling the implant on and off. Someone rewrote the code not knowing what they were doing and broke it.

**Pull request merged**